### ISSUE TITLE
Fonctionnalités sociales / gaming 

### ISSUE THREAD

**Issue by Julie-JDS** [Jardin-des-Sciences/website#13]
Si on propose des logiques interactives inspirées des jeux sociaux, du gaming (voir CCTP) : 
y a t il des modules existants sur Drupal à ce sujet ? 
Si non comment on peut envisager cet aspect ?
Nous n'avons pas une vue fine de ce qui pourrait être envisagé : 
c'est un point dont nous devrons parler également pour le clarifier.

**Comment by bzg** [comment]
Le domaine de la « gamification » est très vaste : cela va de l'énigme que tu décris dans un autre ticket, à des choses plus immersives.  Je n'ai pas de compétence particulière dans le domaine.  Le mieux est pour l'instant d'essayer de trouver une série d'exemples (si possible dans le domaine culturel et patrimonial) dont on pourrait ensuite s'inspirer.

Mais il faut faire vite : car les impacts sur l'ergonomie du site pourront être forts.

**Comment by Julie-JDS** [comment]
L'idéal, le plus juste pour notre positionnement de projet serait d'envisager les sciences participatives comme la dimension de "gaming" / fonctionnalités sociales. Le gaming pur et dur que l'on peut trouver sur mobile, FB, etc, sera / risque d'être gadget. 
Par contre, j'ai besoin d'échanger pour voir comment on pourrait injecter cette dynamique : http://www.operationwardiary.org/?_ga=1.63011247.1213017104.1449242981 ou encore celle ci : http://www.chicagowildlifewatch.org/?_ga=1.63011247.1213017104.1449242981 dans la nôtre. 
Il ne s'agit pas de reprendre la complexité de ses sites qui sont des projets à part entière mais de voir si certaines fonctionnalités interactives pourraient être développées à partir de notre logique interne de projets et prendre place dans nos dossiers... A discuter de vive voix.

**Comment by bzg** [comment]
Oui, à discuter de vive voix, et avant la fin de l'itération 2, car le sujet arrive tard.

Je suis pour une gamification la plus légère possible - mais je vois une manière d'intégrer cela avec l'idée de l'énigme collaborative (voir https://github.com/Jardin-des-Sciences/website/issues/12#issuecomment-161993592).

La question devient : comment faire, de façon légère, pour que les internautes connectés soient incité à ajouter des connaissances sur des fiches pour lesquelles il manque des informations ?

Cela demande de bien réfléchir au rôle que les administrateurs auront dans la sélection des contenus à compléter et dans l'éditorialisation générale du « jeu ».
